opentelemetry/README.md

```diff
@@ -58,11 +58,12 @@ See the [examples](./examples) directory for different integration patterns.
 
 ### Related Crates
 
-In addition to `opentelemetry`, the [`open-telemetry/opentelemetry-rust`]
-repository contains several additional crates designed to be used with the
-`opentelemetry` ecosystem. This includes a collection of trace `SpanExporter`
-and metrics pull and push controller implementations, as well as utility and
-adapter crates to assist in propagating state and instrumenting applications.
+In addition to `opentelemetry`, which only carries the API, the
```

Comment:
Just a thought, if opentelemetry/README.md should now only contain the information specific to API. Most of the content is similar to the root level README.md, and can be removed. Probably something to be addressed once this PR is merged.